tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -806,35 +806,11 @@ show_techsupport/test_techsupport.py::test_techsupport:
 #######################################
 #####            snmp             #####
 #######################################
-snmp/test_snmp_interfaces.py::test_snmp_interfaces:
-  skip:
-    reason: "Interfaces not present on supervisor node."
-    conditions:
-      - "is_supervisor"
-
-snmp/test_snmp_lldp.py:
-  skip:
-    reason: "LLDP not supported on supervisor node."
-    conditions:
-      - "is_supervisor"
-
-snmp/test_snmp_pfc_counters.py:
-  skip:
-    reason: "M0/MX topo does not support test_snmp_pfc_counters"
-    conditions:
-      - "topo_name in ['m0', 'mx']"
```

Comment:
Why remove this skipping for m0 and mx topology?

Reply:
remove this in the latest commit